application/roaddistance/utilities/generate_token.py

```diff
@@ -0,0 +1,25 @@
+import time
+import bcrypt
+
+
```

Comment:
Is this just test code, should it be merged into master?

Reply:
No, this is a utility to generate a token and verify if needed. It does not have the secret included and is just a utility like the bin file generators and stuff

Reply:
Fair enough